mustang_widgets/lib/src/widgets/route_redirect.dart

```diff
@@ -53,6 +58,14 @@ class RouteRedirect extends StatelessWidget {
         ),
       );
     }
-    return child;
+    return Container(
```

Comment:
I think if you allow one more argument, offlineWidget, then it will give more flexibility